kroxylicious-test-tools/pom.xml

```diff
@@ -20,12 +20,33 @@
     </parent>
     <artifactId>kroxylicious-test-tools</artifactId>
     <name>kroxylicious-test-tools</name>
+    <properties>
```

Comment:
jdk 19 is already EOL, so we should bump to 20 already.  I've been meaning to raise a separate PR for this.

Reply:
would it be worth spinning off the resolver to a separate module?  That way kroxylicious-test-tools could remain at JDK 17.  That might make consumption by third parties easier.

Reply:
that's fixed on main now, please just rebase/

Reply:
Thanks!

RE the resolver. I've decided to revert moving the integration tests around and moved the resolver back to `integrationtests`, to put this decision off.